pinot-common/src/main/java/com/linkedin/pinot/common/data/TimeGranularitySpec.java

```diff
@@ -183,10 +183,24 @@ public void setTimeFormat(String timeFormat) {
     this._timeFormat = timeFormat;
   }
 
+  @Deprecated
   public String getTimeFormat() {
```

Comment:
Nice. 
Please, also change the existing calls to getTimeFormat() to use the other calls u have introduced. Also, add a javadoc to getTimeFormat() pointing to the other methods to use.

Reply:
done, will also get a review from @npawar